portfolio/src/main/webapp/style.css

```diff
@@ -3,6 +3,22 @@
   margin-right: auto;
   width: 650px;
 }
+body {
+  background-color: rgb(235, 190, 232)
+}
+img {
```

Comment:
nit: probably better to use the class to define this CSS, instead of applying to all `img` tags.